<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkoutPlanDialog</class>
 <widget class="QDialog" name="WorkoutPlanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Plan name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPlanName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Available exercises:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboExercises"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Exercises in Plan:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listPlanEntries"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnAddExercise">
     <property name="text">
      <string>Add to Plan</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRemoveExercise">
     <property name="text">
      <string>Remove from Plan</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WorkoutPlanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WorkoutPlanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
